output file:
processed_GPflowndiagquad39.json
function:
ndiagquad
Error Cases:
2025-02-13 22:45:29.700578: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:477] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1739504729.712648 1511134 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1739504729.716404 1511134 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-02-13 22:45:29.729535: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.

Pass or Failed: 0

Related Failed Test Cases:
{'../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph FAILED', 'FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10]', '../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10] FAILED', 'FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10]', '../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error FAILED', 'FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error', 'FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10]', '../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10] FAILED', '../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0] FAILED', 'FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0]', 'FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph', '../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10] FAILED'}

All Test Cases On Generated code:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/GPflow/GPflow/venv/bin/python
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/publishablew/GPflow/GPflow
collecting ... collected 6 items

../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0] FAILED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10] FAILED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10] FAILED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10] FAILED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error FAILED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph FAILED

=================================== FAILURES ===================================
__________________________ test_diagquad_1d[var0-mu0] __________________________

mu = array([1. , 1.3]), var = array([3. , 3.5])

    @pytest.mark.parametrize("mu", [np.array([1.0, 1.3])])
    @pytest.mark.parametrize("var", [np.array([3.0, 3.5])])
    def test_diagquad_1d(mu: TensorType, var: TensorType) -> None:
        num_gauss_hermite_points = 25
>       quad = quadrature.ndiagquad([lambda *X: tf.exp(X[0])], num_gauss_hermite_points, [mu], [var])

/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:71: in ndiagquad
    from .temp import ndiagquad
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import itertools
    import warnings
    from functools import wraps
    from typing import Callable, Iterable, List, Optional, Tuple, Union
    import numpy as np
    import tensorflow as tf
    from check_shapes import check_shapes
    from ..base import AnyNDArray, TensorType
    from ..config import default_float
    from ..utilities import to_default_float
    from .gauss_hermite import NDiagGHQuadrature
    import warnings
    import tensorflow as tf
    from typing import Callable, Iterable, Union, Tuple, List
    from gpflow.quadrature import NDiagGHQuadrature
    from gpflow.base import TensorType
>   from gpflow.utilities import check_shapes
E   ImportError: cannot import name 'check_shapes' from 'gpflow.utilities' (/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/utilities/__init__.py)

/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/temp.py:17: ImportError
___________________ test_diagquad_2d[var20-mu20-var10-mu10] ____________________

mu1 = array([1. , 1.3]), var1 = array([3. , 3.5]), mu2 = array([-2. ,  0.3])
var2 = array([4. , 4.2])

    @pytest.mark.parametrize("mu1", [np.array([1.0, 1.3])])
    @pytest.mark.parametrize("var1", [np.array([3.0, 3.5])])
    @pytest.mark.parametrize("mu2", [np.array([-2.0, 0.3])])
    @pytest.mark.parametrize("var2", [np.array([4.0, 4.2])])
    def test_diagquad_2d(mu1: TensorType, var1: TensorType, mu2: TensorType, var2: TensorType) -> None:
        alpha = 2.5
        # using logspace=True we can reduce this, see test_diagquad_logspace
        num_gauss_hermite_points = 35
>       quad = quadrature.ndiagquad(
            lambda *X: tf.exp(X[0] + alpha * X[1]),
            num_gauss_hermite_points,
            [mu1, mu2],
            [var1, var2],
        )

/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:43: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:71: in ndiagquad
    from .temp import ndiagquad
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import itertools
    import warnings
    from functools import wraps
    from typing import Callable, Iterable, List, Optional, Tuple, Union
    import numpy as np
    import tensorflow as tf
    from check_shapes import check_shapes
    from ..base import AnyNDArray, TensorType
    from ..config import default_float
    from ..utilities import to_default_float
    from .gauss_hermite import NDiagGHQuadrature
    import warnings
    import tensorflow as tf
    from typing import Callable, Iterable, Union, Tuple, List
    from gpflow.quadrature import NDiagGHQuadrature
    from gpflow.base import TensorType
>   from gpflow.utilities import check_shapes
E   ImportError: cannot import name 'check_shapes' from 'gpflow.utilities' (/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/utilities/__init__.py)

/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/temp.py:17: ImportError
________________ test_diagquad_logspace[var20-mu20-var10-mu10] _________________

mu1 = array([1. , 1.3]), var1 = array([3. , 3.5]), mu2 = array([-2. ,  0.3])
var2 = array([4. , 4.2])

    @pytest.mark.parametrize("mu1", [np.array([1.0, 1.3])])
    @pytest.mark.parametrize("var1", [np.array([3.0, 3.5])])
    @pytest.mark.parametrize("mu2", [np.array([-2.0, 0.3])])
    @pytest.mark.parametrize("var2", [np.array([4.0, 4.2])])
    def test_diagquad_logspace(
        mu1: TensorType, var1: TensorType, mu2: TensorType, var2: TensorType
    ) -> None:
        alpha = 2.5
        num_gauss_hermite_points = 25
>       quad = quadrature.ndiagquad(
            lambda *X: (X[0] + alpha * X[1]),
            num_gauss_hermite_points,
            [mu1, mu2],
            [var1, var2],
            logspace=True,
        )

/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:62: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:71: in ndiagquad
    from .temp import ndiagquad
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import itertools
    import warnings
    from functools import wraps
    from typing import Callable, Iterable, List, Optional, Tuple, Union
    import numpy as np
    import tensorflow as tf
    from check_shapes import check_shapes
    from ..base import AnyNDArray, TensorType
    from ..config import default_float
    from ..utilities import to_default_float
    from .gauss_hermite import NDiagGHQuadrature
    import warnings
    import tensorflow as tf
    from typing import Callable, Iterable, Union, Tuple, List
    from gpflow.quadrature import NDiagGHQuadrature
    from gpflow.base import TensorType
>   from gpflow.utilities import check_shapes
E   ImportError: cannot import name 'check_shapes' from 'gpflow.utilities' (/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/utilities/__init__.py)

/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/temp.py:17: ImportError
_____________________ test_diagquad_with_kwarg[var10-mu10] _____________________

mu1 = array([1. , 1.3]), var1 = array([3. , 3.5])

    @pytest.mark.parametrize("mu1", [np.array([1.0, 1.3])])
    @pytest.mark.parametrize("var1", [np.array([3.0, 3.5])])
    def test_diagquad_with_kwarg(mu1: TensorType, var1: TensorType) -> None:
        alpha: AnyNDArray = np.array([2.5, -1.3])
        num_gauss_hermite_points = 25
>       quad = quadrature.ndiagquad(
            lambda X, Y: tf.exp(X * Y), num_gauss_hermite_points, mu1, var1, Y=alpha
        )

/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:78: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:71: in ndiagquad
    from .temp import ndiagquad
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import itertools
    import warnings
    from functools import wraps
    from typing import Callable, Iterable, List, Optional, Tuple, Union
    import numpy as np
    import tensorflow as tf
    from check_shapes import check_shapes
    from ..base import AnyNDArray, TensorType
    from ..config import default_float
    from ..utilities import to_default_float
    from .gauss_hermite import NDiagGHQuadrature
    import warnings
    import tensorflow as tf
    from typing import Callable, Iterable, Union, Tuple, List
    from gpflow.quadrature import NDiagGHQuadrature
    from gpflow.base import TensorType
>   from gpflow.utilities import check_shapes
E   ImportError: cannot import name 'check_shapes' from 'gpflow.utilities' (/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/utilities/__init__.py)

/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/temp.py:17: ImportError
_____________________ test_ndiagquad_does_not_throw_error ______________________

    def test_ndiagquad_does_not_throw_error() -> None:
        """
        Check that the autograph=False for quadrature.ndiagquad does not throw an error.
        Regression test for https://github.com/GPflow/GPflow/issues/1547.
        """
    
        @tf.function(autograph=False)
        def func_ndiagquad_autograph_false() -> tf.Tensor:
            mu: AnyNDArray = np.array([1.0, 1.3])
            var: AnyNDArray = np.array([3.0, 3.5])
            num_gauss_hermite_points = 25
            return quadrature.ndiagquad(
                [lambda *X: tf.exp(X[0])], num_gauss_hermite_points, [mu], [var]
            )
    
>       func_ndiagquad_autograph_false()

/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:100: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow/python/util/traceback_utils.py:153: in error_handler
    raise e.with_traceback(filtered_tb) from None
/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:96: in func_ndiagquad_autograph_false
    return quadrature.ndiagquad(
/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:71: in ndiagquad
    from .temp import ndiagquad
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import itertools
    import warnings
    from functools import wraps
    from typing import Callable, Iterable, List, Optional, Tuple, Union
    import numpy as np
    import tensorflow as tf
    from check_shapes import check_shapes
    from ..base import AnyNDArray, TensorType
    from ..config import default_float
    from ..utilities import to_default_float
    from .gauss_hermite import NDiagGHQuadrature
    import warnings
    import tensorflow as tf
    from typing import Callable, Iterable, Union, Tuple, List
    from gpflow.quadrature import NDiagGHQuadrature
    from gpflow.base import TensorType
>   from gpflow.utilities import check_shapes
E   ImportError: cannot import name 'check_shapes' from 'gpflow.utilities' (/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/utilities/__init__.py)

/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/temp.py:17: ImportError
__________________________ test_quadrature_autograph ___________________________

    def test_quadrature_autograph() -> None:
        """
        Check that the return value is equal with and without Autograph
        Regression test for https://github.com/GPflow/GPflow/issues/1547.
        """
    
        def compute(autograph: bool) -> AnyNDArray:
            @tf.function(autograph=autograph)
            def func() -> tf.Tensor:
                mu: AnyNDArray = np.array([1.0, 1.3])
                var: AnyNDArray = np.array([3.0, 3.5])
                num_gauss_hermite_points = 25
                return quadrature.ndiagquad(
                    [lambda *X: tf.exp(X[0])], num_gauss_hermite_points, [mu], [var]
                )
    
            (result,) = func()
            return cast(AnyNDArray, result.numpy())
    
        np.testing.assert_equal(
>           compute(autograph=True),
            compute(autograph=False),
        )

/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:123: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py:119: in compute
    (result,) = func()
/local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow/python/util/traceback_utils.py:153: in error_handler
    raise e.with_traceback(filtered_tb) from None
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = (), kwargs = {}

    def autograph_handler(*args, **kwargs):
      """Calls a converted version of original_func."""
      try:
        return api.converted_call(
            original_func,
            args,
            kwargs,
            options=converter.ConversionOptions(
                recursive=True,
                optional_features=autograph_options,
                user_requested=True,
            ))
      except Exception as e:  # pylint:disable=broad-except
        if hasattr(e, "ag_error_metadata"):
>         raise e.ag_error_metadata.to_exception(e)
E         tensorflow.python.autograph.impl.api.StagingError: in user code:
E         
E             File "/local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py", line 116, in func  *
E                 [lambda *X: tf.exp(X[0])], num_gauss_hermite_points, [mu], [var]
E             File "/local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/check_shapes/decorator.py", line 69, in wrapped_function  *
E                 def wrapped_function(*args: Any, **kwargs: Any) -> Any:
E             File "/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py", line 71, in ndiagquad  *
E                 from .temp import ndiagquad
E             File "/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/temp.py", line 17, in <module>
E                 from gpflow.utilities import check_shapes
E         
E             ImportError: cannot import name 'check_shapes' from 'gpflow.utilities' (/local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/utilities/__init__.py)

/local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow/python/eager/polymorphic_function/autograph_util.py:52: StagingError
=============================== warnings summary ===============================
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:57
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:57: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if (distutils.version.LooseVersion(tf.__version__) <

../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:58
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:58: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    distutils.version.LooseVersion(required_tensorflow_version)):

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0]
FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10]
FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10]
FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10]
FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error
FAILED ../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph
======================== 6 failed, 2 warnings in 1.82s =========================


Final Test Result:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/GPflow/GPflow/venv/bin/python
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/publishablew/GPflow/GPflow
collecting ... collected 6 items

../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph PASSED

=============================== warnings summary ===============================
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:57
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:57: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if (distutils.version.LooseVersion(tf.__version__) <

../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:58
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:58: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    distutils.version.LooseVersion(required_tensorflow_version)):

tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0]
tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10]
tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10]
tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10]
tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error
tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph
  /local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:165: DeprecationWarning: Please use gpflow.quadrature.NDiagGHQuadrature instead (note the changed convention of how multi-dimensional quadrature is handled)
    warnings.warn(

tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow/python/autograph/impl/api.py:371: DeprecationWarning: Please use gpflow.quadrature.NDiagGHQuadrature instead (note the changed convention of how multi-dimensional quadrature is handled)
    return py_builtins.overload_of(f)(*args)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 6 passed, 9 warnings in 2.72s =========================


Initial Result:
============================= test session starts ==============================
platform linux -- Python 3.11.10, pytest-8.3.4, pluggy-1.5.0 -- /local/data0/moved_data/publishablew/GPflow/GPflow/venv/bin/python
cachedir: .pytest_cache
rootdir: /local/data0/moved_data/publishablew/GPflow/GPflow
collecting ... collected 6 items

../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10] PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error PASSED
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph PASSED

=============================== warnings summary ===============================
../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:57
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:57: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if (distutils.version.LooseVersion(tf.__version__) <

../../../../../../local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:58
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow_probability/python/__init__.py:58: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    distutils.version.LooseVersion(required_tensorflow_version)):

tests/gpflow/quadrature/test_quadrature.py::test_diagquad_1d[var0-mu0]
tests/gpflow/quadrature/test_quadrature.py::test_diagquad_2d[var20-mu20-var10-mu10]
tests/gpflow/quadrature/test_quadrature.py::test_diagquad_logspace[var20-mu20-var10-mu10]
tests/gpflow/quadrature/test_quadrature.py::test_diagquad_with_kwarg[var10-mu10]
tests/gpflow/quadrature/test_quadrature.py::test_ndiagquad_does_not_throw_error
tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph
  /local/data0/moved_data/publishablew/GPflow/GPflow/gpflow/quadrature/deprecated.py:165: DeprecationWarning: Please use gpflow.quadrature.NDiagGHQuadrature instead (note the changed convention of how multi-dimensional quadrature is handled)
    warnings.warn(

tests/gpflow/quadrature/test_quadrature.py::test_quadrature_autograph
  /local/data0/moved_data/publishablew/GPflow/GPflow/venv/lib/python3.11/site-packages/tensorflow/python/autograph/impl/api.py:371: DeprecationWarning: Please use gpflow.quadrature.NDiagGHQuadrature instead (note the changed convention of how multi-dimensional quadrature is handled)
    return py_builtins.overload_of(f)(*args)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 6 passed, 9 warnings in 2.73s =========================
